src/FeedForwardPid.cpp

```diff
@@ -6,29 +6,81 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#include <cmath>
+#include <limits>
+#include <ros/ros.h>
+
 #include "iarc7_motion/FeedForwardPid.hpp"
 
 using namespace Iarc7Motion;
 
-FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain) : p_gain_(p_gain), i_gain_(i_gain), d_gain_(d_gain), setpoint_(0.0)
+FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain,
+                               double i_accumulator_max, double i_accumulator_min)
+    : p_gain_(p_gain),
+      i_gain_(i_gain),
+      d_gain_(d_gain),
+      initialized_(false),
+      i_accumulator_(0.0),
+      last_current_value_(0.0),
+      setpoint_(0.0),
+      i_accumulator_max_(i_accumulator_max),
+      i_accumulator_min_(i_accumulator_min)
 {
     // Nothing to do
 }
 
-double FeedForwardPid::update(double current_value, double time_delta)
+bool FeedForwardPid::update(double current_value, double time_delta, double& response)
```

Comment:
I think the cleanest way would be to call this with a time instead of a delta, and then it's called with the same information every time.  The only problem with that is if you were trying to keep this class independent of ROS